docs/policy/SecurityPolicyManual.md

```
@@ -342,12 +345,14 @@ The `theSupervisedCodeUsesTheFollowingPackage` field specifies the root package
 **Examples:**
 
 Using a shallow package prefix:
+
 ```yaml
 theSupervisedCodeUsesTheFollowingPackage: "de.tum.cit.aet"
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)